persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -42,32 +41,18 @@ private PersistToolsConstants() {}
     public static final String KEY_PASSWORD = "password";
     public static final String KEY_DATABASE = "database";
     public static final String KEY_HOST = "host";
-    public static final String KEY_PROVIDER = "provider";
 
     public static final String CONFIG_SCRIPT_FILE = "Config.toml";
     public static final String PASSWORD = "password";
     public static final String USER = "user";
     public static final String MYSQL_DRIVER_CLASS = "com.mysql.cj.jdbc.Driver";
 
-    public static final String DATABASE = "database";
-    public static final String DATABASE_MYSQL = "database.mysql";
-
-    public static final String AUTO_INCREMENT_WITH_SPACE = " AUTO_INCREMENT";
-    public static final String AUTO_INCREMENT_WITH_TAB = "  AUTO_INCREMENT";
-    public static final String START_VALUE = "startValue";
-    public static final String INCREMENT = "increment";
-    public static final String ON_UPDATE = "onUpdate";
-    public static final String ON_DELETE = "onDelete";
-    public static final String CONSTRAINT = "constraint";
-    public static final String STRING = "String";
-    public static final String LENGTH = "length";
-    public static final String MAX_LENGTH = "maxLength";
-    public static final String SQL_SCHEMA_FILE = "persist_db_scripts.sql";
-    public static final String UNIQUE_CONSTRAINTS = "uniqueConstraints";
+    public static final String PERSIST_CONFIG_PATTERN = "persist.%s.storage";
+    public static final String PERSIST_CONFIG_PATTERN_WITH_MYSQL = "persist.%s.storage.mysql";
+    public static final String SQL_SCHEMA_FILE = "%s_db_scripts.sql";
     public static final String PERSIST_DIRECTORY = "persist";
     public static final String GENERATED_DIRECTORY = "generated";
-    public static final String PERSIST_TOML_FILE = "Persist.toml";
-    public static final String PATH_BALLERINA_TOML = "Ballerina.toml";
+    public static final String BALLERINA_TOML_FILE = "Ballerina.toml";
```

Comment:
Hall we use the one from ballerina-lang ProjectAPIConstants?

Reply:
Fixed in [83a51f2](https://github.com/ballerina-platform/persist-tools/pull/101/commits/83a51f236d9f746e16b46047d40281ecd05ba7fe)